src/components/markdown-renderer/markdown-extension/link-replacer/jump-anchor.tsx

```diff
@@ -14,7 +14,12 @@ export interface JumpAnchorProps extends AllHTMLAttributes<HTMLAnchorElement> {
 export const JumpAnchor: React.FC<JumpAnchorProps> = ({ jumpTargetId, children, ...props }) => {
   const jumpToTargetId = useCallback(
     (event: React.MouseEvent<HTMLElement, MouseEvent>): void => {
-      document.getElementById(jumpTargetId)?.scrollIntoView({ behavior: 'smooth' })
+      const intoViewElement = document.getElementById(jumpTargetId)
```

Comment:
Maybe we should add a todo or comment here explaining why we don't use scrollIntoView?

Reply:
done